lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -39,16 +39,20 @@ def initialize_project(project)
       @kit = @platform['kit']
     end
 
+    def studio_vm_options
+      {
+        'id' => @id.to_i,
+        'client_id' => 0,
+        'workspace_path' => @workspace_path,
+        'image_name' => @platform['st_image'],
+        'logger' => @logger,
+        'slirp' => @slirp,
+        'iso_path' => @project.config['iso_path']
+      }.merge(platform_options)
```

Comment:
I am okay with these changes until HLK Studio can successfully be installed with an enabled secure boot. Currently, there is no limitation.